tests/unit/type/string.tcl

```diff
@@ -459,6 +459,17 @@ start_server {tags {"string"}} {
             assert_equal [string range $bin $_start $_end] [r getrange bin $start $end]
         }
     }
+
+    test "SUBSTR fuzzing" {
```

Comment:
i think we just want a basic test for coverage.. no need for fuzzing

Reply:
Looks like SUBSTR is deprecated in favor of GETRANGE. So, I copied this test from the just above GETRANGE test. 
Anyway, changed it with some basic test.